src/Data/List/Relation/Binary/Pointwise/Base.agda

```diff
@@ -8,16 +8,16 @@
 
 module Data.List.Relation.Binary.Pointwise.Base where
 
-open import Data.Product.Base using (_×_; <_,_>)
+open import Data.Product.Base as Σ using (_×_; _,_; <_,_>; ∃-syntax)
```

Comment:
Do we leave this, or bring it into line with #2280 / #2284 ?

Reply:
Fixed both.